features/src/main/feature/camel-features.xml

```diff
@@ -1118,23 +1118,26 @@
-<!--        <bundle dependency='true'>mvn:org.apache.httpcomponents/httpclient-osgi/${httpclient4-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:org.apache.httpcomponents/httpasyncclient-osgi/${httpasyncclient-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:org.codehaus.jettison/jettison/${jettison-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:joda-time/joda-time/${jodatime2-version}</bundle>-->
-<!--        <bundle dependency='true'>wrap:mvn:com.google.code.gson/gson/${gson-version}</bundle>-->
-<!--        <bundle dependency='true'>wrap:mvn:io.opencensus/opencensus-api/${google-http-client-opencensus-api-version}</bundle>-->
-<!--        <bundle dependency='true'>wrap:mvn:io.opencensus/opencensus-contrib-http-util/${google-http-client-opencensus-api-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.google-http-client/${google-http-client-bundle-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.guava/${jira-guava-bundle-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.atlassian-jira-client/${atlassian-jira-client-bundle-version}</bundle>-->
-<!--        <bundle>mvn:org.apache.camel/camel-jira/${project.version}</bundle>-->
-<!--    </feature>-->
+    <feature name='camel-jira' version='${project.version}' start-level='50'>
+        <feature version='${camel.osgi.version.range}'>camel-core</feature>
+        <feature version='[33,34)'>guava</feature>
+        <feature version='4.5.14'>http-client</feature>
+        <bundle dependency='true'>wrap:mvn:com.atlassian.event/atlassian-event/4.1.3</bundle>
+        <bundle dependency='true'>wrap:mvn:com.atlassian.httpclient/atlassian-httpclient-library/2.1.5</bundle>
+        <bundle dependency='true'>wrap:mvn:com.atlassian.httpclient/atlassian-httpclient-api/2.1.5</bundle>
+        <bundle dependency='true'>wrap:mvn:com.atlassian.jira/jira-rest-java-client-core/${jira-rest-client-api-version}</bundle>
+        <bundle dependency='true'>wrap:mvn:com.atlassian.jira/jira-rest-java-client-api/${jira-rest-client-api-version}</bundle>
+        <bundle dependency='true'>wrap:mvn:com.atlassian.sal/sal-api/5.1.4$overwrite=merge&amp;Import-Package=*;resolution:=optional&amp;Export-Package=com.atlassian.sal.api*;version=5.1.4</bundle>
+        <bundle dependency='true'>mvn:io.atlassian.util.concurrent/atlassian-util-concurrent/4.0.1</bundle>
+        <bundle dependency='true'>wrap:mvn:com.google.http-client/google-http-client/1.43.3$overwrite=merge&amp;Import-Package=io.opencensus*,com.google.api.client*,org.apache.http*,javax*,com.google.common*;version="[33,34)"&amp;Export-Package=com.google.api.client.auth*;version=1.43.3</bundle>
+        <bundle dependency='true'>wrap:mvn:com.google.http-client/google-http-client-gson/1.43.3</bundle>
```

Comment:
it compiles without this dependency

Reply:
Broadened the import to com.google.api.client*  ending up with
`<bundle dependency='true'>wrap:mvn:com.google.oauth-client/google-oauth-client/1.35.0$overwrite=merge&amp;Import-Package=com.google.api.client*,javax*,com.google.common*;version="[33,34)"&amp;Export-Package=com.google.api.client*;version=1.35.0</bundle>`

to comply with the manifest

`Import-Package: com.google.api.client.auth.oauth2,com.google.api.clien
 t.http;version="[1.43,2)",com.google.api.client.http.javanet;version=
 "[1.43,2)",com.google.api.client.json;version="[1.43,2)",com.google.a
 pi.client.json.gson,com.google.api.client.json.webtoken;version="[1.4
 3,2)",com.google.api.client.util;version="[1.43,2)",com.google.api.cl
 ient.util.escape;version="[1.43,2)",com.google.api.client.util.store;
 version="[1.43,2)",com.google.common.base;version="[31.1,32)",com.goo
 gle.common.cache;version="[31.1,32)",com.google.common.collect;versio
 n="[31.1,32)",com.google.common.io;version="[31.1,32)",com.google.com
 mon.util.concurrent;version="[31.1,32)",javax.crypto,javax.crypto.spe
 c`

 so replaced com.google.api.client.auth* with com.google.api.client*
then the gson dependencies are needed